docs/reference/cli/cmd_chunk.md

```diff
@@ -118,6 +118,18 @@ This feature is not yet implemented.
 
 ## Preventing Possible Correctness Issues
 
+It is theoretically possible to detect nearly all of the sources of correctness
+issues and prevent rules from forming when those situations are detected. In
+Soar 9.6.0, though, only one filter is available, `allow-local-negations`. Future
+versions of Soar will include more correctness filters.
+
+Note that it is still possible to detect that your agent may have encountered a
+known source of a correctness issue by looking at the output of the chunk
+stats command. It has specific statistics for some of the sources, while others
```

Comment:
I think instead of just "chunk stats" this should state the specific command for the user to try.

Reply:
Since you didn't write it, I don't think it's fair to ask you to fix it, so you can ignore this and other similar comments.